test/fuzz/gen.ml

```diff
@@ -173,6 +211,10 @@ let rec expr ~block_type ~stack ~locals env =
     let i = const i in
     list_cons i next
 
+let memory env =
+  let+ id = const @@ Env.add_memory env in
+  MMem (Some id, { min = 0; max = None})
```

Comment:
We should have a random min (but not too big !) and generate an (int option) for the max.

Reply:
Mémo : warning, time explosion during fuzzing if memory size is fully random.
(constrained size for now)

Reply:
Could you add a TODO with a link to https://github.com/OCamlPro/owi/pull/28#discussion_r1212835761 ?